arch/arm/core/cortex_m/Kconfig

```diff
@@ -203,7 +203,8 @@ config SW_VECTOR_RELAY
 	prompt "Enable Cortex-M0 Vector Table soft relay"
 	default n
 	default y if BOOTLOADER_MCUBOOT
-	depends on CPU_CORTEX_M0 && !CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP
+	depends on CPU_CORTEX_M0 && !(CPU_CORTEX_M_HAS_VTOR ||
```

Comment:
@carlescufi please clarify: can the M0 have a VTOR register, only is this part of M0+?